### PR TITLE
update spark and scikit-learn examples in readme(s)

### PR DESCRIPTION
**Pull request by ancasarb**

### CODE REVIEW

File: mleap-spark/src/main/resources/spark-demo.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+test_string,test_double
```
Comment (hollinwilkins) [inline mleap-spark/src/main/resources/spark-demo.csv]: We probably shouldn't include this with the `mleap-spark` module. Can we create a folder next to `mleap-spark` called `examples` and add it here?

File: README.md [hunk 2]
```diff
@@ -111,39 +111,46 @@ The first step is to create our pipeline in Spark. For our example we will manua
...
+import org.apache.spark.ml.feature.{Binarizer, StringIndexer}
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+import resource._
 
-// User out-of-the-box Spark transformers like you normally would
-val stringIndexer = new StringIndexerModel(uid = "si", labels = Array("hello", "MLeap")).
-  setInputCol("test_string").
-  setOutputCol("test_index")
+  val datasetName = "./mleap-docs/assets/spark-demo.csv"
```
Comment (hollinwilkins) [inline README.md]: This should probably point to the `examples` folder where we add the spark-demo.csv file.

Comment (ancasarb) [reply README.md]: Updated.

File: mleap-spark/src/main/resources/spark-demo.csv (hunk 1 above)
Comment (ancasarb) [reply mleap-spark/src/main/resources/spark-demo.csv]: Yep, good point, I intended to put it under test/resources. I've created the examples folder as you suggested instead.

**Pull request merged**